src/99-appendices/03-hed.md

```diff
@@ -1,63 +1,71 @@
 # Appendix III: Hierarchical Event Descriptors
 
-Hierarchical Event Descriptors (HED) are a controlled vocabulary of terms describing events in a behavioral
-paradigm. HED was originally developed with EEG in mind, but is applicable to
-all behavioral experiments. Each level of a hierarchical tag is delimited with
-a forward slash (`/`). A HED string contains one or more HED tags separated by
-commas (`,`). Parentheses (brackets, `()`) group tags and enable specification
-of multiple items and their attributes in a single HED string (see section 2.4
-in [HED Tagging Strategy Guide](http://www.hedtags.org/downloads/HED%20Tagging%20Strategy%20Guide.pdf)).
+Hierarchical Event Descriptors (HED) are a controlled vocabulary of terms describing
+events in a machine-actionable form so that algorithms can use the information without
+manual recoding. HED was originally developed with EEG in mind, but is applicable to
+all behavioral experiments.
+
+Each level of a hierarchical tag is delimited with a forward slash (`/`). A HED string
+contains one or more HED tags separated by commas (`,`). Parentheses (brackets, `()`)
+group tags and enable specification of multiple items and their attributes in a
+single **HED string** (see section 2.4 in
+[HED Tagging Strategy Guide](http://www.hedtags.org/downloads/HED%20Tagging%20Strategy%20Guide.pdf)).
 For more information about HED and tools available to validate and match HED
 strings, please visit [www.hedtags.org](http://www.hedtags.org). Since
 dedicated fields already exist for the overall task classification in the
 sidecar JSON files (`CogAtlasID` and `CogPOID`), HED tags from the `Paradigm`
 HED subcategory should not be used to annotate events.
 
+## Annotating each event
+
 There are several ways to associate HED annotations with events within the BIDS
 framework. The most direct way is to use the `HED` column of the `*_events.tsv`
-file to annotate events:  
+file to annotate events.
 
-Example:
+Example: An `*_events.tsv` annotated using HED tags for individual events.
 
 ```Text
-onset	duration	HED
-1.1	n/a	Event/Category/Experimental stimulus, Event/Label/CrossFix,  Sensory presentation/Visual, Item/Object/2D Shape/Cross
-1.3	n/a	Event/Category/Participant response, Event/Label/ButtonPress, Action/Button press
+onset  duration  HED
+1.1    n/a       Event/Category/Experimental stimulus, Event/Label/CrossFix,  Sensory presentation/Visual, Item/Object/2D Shape/Cross
+1.3    n/a       Event/Category/Participant response, Event/Label/ButtonPress, Action/Button press
```

Comment:
here you replaced the "tab" character `	` with a bunch of "space" characters. 

I know that we have both variants in the specficiation. The "tab" character style of formatting the TSV examples has the advantage that users can simply copy and paste it ... at the expense that sometimes the columns are not aligned.

I am not sure what kind of style we should enforce throughout the spec, but consistency would probably pay off. Opinions @tsalo @effigies ? I know that @yarikoptic always wants "tab" characters.

Reply:
Apologies on the line breaks---I'll try to remember for next time.  I ended up changing the tabs to spaces because I found the table unreadable with the columns not aligned.  This is problematic for more complex tables.